acceptance/Rakefile

```diff
@@ -1,31 +1,37 @@
 require 'rake/clean'
 
+# Default test target if none specified
```

Comment:
@ajroetker based on a conversation with @zreichert I believe you will be modifying this PR to remove any changes the Rakefile and that rake updates will be handled separately.